tests/v2/validation/charts/backup_restore/README.md

```diff
@@ -0,0 +1,32 @@
+# Backup Restore Operator (BRO)
+Backup Restore Operator (BRO for short) is a disaster recovery chart that can be installed on the local cluster only, and is used to create backups of Rancher resources contained in the `rancher-resource-set`. The resource set becomes available after the chart is installed and can be edited as a user needs. Once a backup is created and stored in either a local volume or in an S3 storage location (such as AWS S3 or Minio S3), a restore operation can be created to restore Rancher back to the backed up version of the Rancher resources.
+
+## Pre-requisites
+- Downstream RKE1 and RKE2 clusters are required before running the test(s)
```

Comment:
At least one of each, but now I'm looking into a way to maybe take in a list of cluster names from the config so that how many doesn't necessarily matter

Reply:
after conversations with Caleb, Israel, William, and BRO devs I'll be provisioning the clusters in the test itself to make the test more self-sufficient.